todo_app/Ansible/my-ansible-playbook.yml

```diff
@@ -0,0 +1,78 @@
+- name: Install todo app on new web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: api_key
+      prompt: What is the Trello API key?
+
+    - name: token
+      prompt: What is the Trello Token?
+
+    - name: board_id
+      prompt: What is the Trello board id?
+
+    - name: todo_list_id
+      prompt: What is the to do list id?
+
+    - name: started_list_id
+      prompt: What is the started list id?
+
+    - name: done_list_id
+      prompt: What is the done list id?
+
+  tasks:
+  - name: Install git
+    yum:
+      name: git
+      state: latest
+    become: yes
+
+  - name: Install Python3
+    yum:
+      name: python3
+      state: latest
+    become: yes
+
+  - name: Install poetry
+    ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
+    args:
+      creates: ~/.local/bin/poetry
+
+  - name: Create app folder
+    ansible.builtin.file:
+      path: /var/todo_app
+      state: directory
+      mode: '777'
```

Comment:
This works, but it's an easy and bad habit to fall into setting '777' by default just to ensure you don't have to think about exactly what permissions are appropriate - having permissions be too generous is a common security concern, so for real projects worth revisiting this after getting it working to think about a more locked down setting - '755' is a common one for directories